lib/CSSStyleDeclaration.js

```diff
@@ -17,7 +17,10 @@ function parseStyles(s) {
   parser.addListener("property", function(e) {
     if (e.invalid) return; // Skip errors
     result[e.property.text] = e.value.text;
-    if (e.important) result.important[e.property.text] = e.important;
+    if (e.important) {
+      result.important = {};
```

Comment:
Shouldn't this be:

```js
result.important = result.important || {};
```

to avoid rewriting the object in case it exists already?